### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode' - torrent_metadata

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#4942]
From reporter - Tribler 7.4.0
```python
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 114, in on_read_data
RuntimeError: [Failure instance: Traceback: <class 'AttributeError'>: 'str' object has no attribute 'decode'
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:149:render_GET
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:135:on_magnet
lib\site-packages\twisted\internet\defer.py:322:addCallback
lib\site-packages\twisted\internet\defer.py:311:addCallbacks
--- <exception caught here> ---
lib\site-packages\twisted\internet\defer.py:654:_runCallbacks
Tribler\Core\Modules\restapi\torrentinfo_endpoint.py:80:on_got_metainfo
Tribler\Core\Modules\MetadataStore\OrmBindings\torrent_metadata.py:49:tdef_to_metadata_dict
]
```

**Comment by ichorid** [comment]
Happens on creating a torrent from a file:
```python
Traceback (most recent call last):
  File "/home/vader/src/TRIBLER/Tribler/TriblerGUI/dialogs/createtorrentdialog.py", line 132, in on_torrent_created
    self.add_torrent_to_channel(result['torrent'])
  File "/home/vader/src/TRIBLER/Tribler/TriblerGUI/dialogs/createtorrentdialog.py", line 140, in add_torrent_to_channel
    data.update({"title": self.name.decode('utf-8')})
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by ichorid** [comment]
Fixed in #4966
